types/winston/index.d.ts

```diff
@@ -378,32 +376,36 @@ declare namespace winston {
         Webhook: WebhookTransportInstance;
     }
 
-    export type TransportOptions = ConsoleTransportOptions | DailyRotateFileTransportOptions | FileTransportOptions | HttpTransportOptions | MemoryTransportOptions | WebhookTransportOptions | WinstonModuleTransportOptions;
+    type TransportOptions = ConsoleTransportOptions | DailyRotateFileTransportOptions | FileTransportOptions
+        | HttpTransportOptions | MemoryTransportOptions | WebhookTransportOptions | WinstonModuleTransportOptions;
 
-    export interface GenericTransportOptions {
+    interface GenericTransportOptions {
         level?: string;
         silent?: boolean;
         raw?: boolean;
         name?: string;
-        formatter?: Function;
         handleExceptions?: boolean;
         exceptionsLevel?: string;
         humanReadableUnhandledException?: boolean;
+
+        formatter?(options?: any): string;
     }
 
-    export interface GenericTextTransportOptions {
+    interface GenericTextTransportOptions {
         json?: boolean;
         colorize?: boolean;
         colors?: any;
         prettyPrint?: boolean;
-        timestamp?: (Function | boolean);
         showLevel?: boolean;
         label?: string;
         depth?: number;
-        stringify?: Function;
+
+        // tslint:disable-next-line
```

Comment:
Thanks, I fixed this per your suggestion (yes, was trying to avoid prefer-method-signature)!